config/install/user.role.editor.yml

```diff
@@ -0,0 +1,132 @@
+uuid: c23ccd9c-e9f5-4f75-bea1-029eff8e81be
```

Comment:
Remove this file. This file already exists here - https://github.com/srijanone/ezcontent/blob/8.x/modules/ezcontent_user/config/install/user.role.editor.yml
Is there a change that needs to be pushed ?